src/GoogleAuthenticator.php

```diff
@@ -86,6 +86,21 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
+    final protected function setPeriod($period):bool
```

Comment:
Can you add a type hint to the parameter?

Reply:
And afterwards the null check can be removed